cartridges/int_alma/cartridge/scripts/steps/CapturePaymentOrders.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var OrderMgr = require('dw/order/OrderMgr');
+var Status = require('dw/system/Status');
+var almaOrderHelper = require('*/cartridge/scripts/helpers/almaOrderHelper');
+var almaPaymentHelper = require('*/cartridge/scripts/helpers/almaPaymentHelper');
+var Logger = require('dw/system/Logger');
+
+exports.execute = function () {
+    var orders = OrderMgr.searchOrders("custom.ALMA_Deferred_Capture='toCapture' and status != 8 and status != 6", null);
+    var errors = [];
+
+    if (orders.count > 0) {
+        while (orders.hasNext()) {
+            var order = orders.next();
+            var params = { external_id: order.custom.almaPaymentId };
+
+            try {
+                var capture = almaPaymentHelper.capturePayment(params);
+                almaOrderHelper.setAlmaDeferredCapture(order, capture.id);
```

Comment:
I'm not sure to understand what ALMA_Deferred_Capture contains.
'toCapture' or the capture's id ?
Could it be explicit somewhere ?

Reply:
Both : toCapture and capture ID after the capture.